lib/dialect/postgres.ts

```diff
@@ -392,9 +392,11 @@ export class Postgres extends Dialect {
         const paramNodes = insertNode.getParameters();
         if (paramNodes.length > 0) {
             const paramText = paramNodes
-                .map((paramSet) => {
-                    return paramSet.map((param) => this.visit(param)).join(', ');
-                })
+                .map((paramSet) => paramSet.map((param) => {
+                    const value = this.visit(param).toString()
+                    if (value.startsWith('SELECT')) return `(${value})`
```

Comment:
This solution is very naïve. There may be a more elegant one to deal with all kind of expression but I do not have sufficient knowledge over the library. Please tell me if you see a better option.